webapi/src/main/scala/org/knora/webapi/responders/admin/ListsResponderADM.scala

```diff
@@ -55,11 +55,11 @@ class ListsResponderADM(responderData: ResponderData) extends Responder(responde
       listNodeInfoGetRequestADM(listIri, requestingUser)
     case NodePathGetRequestADM(iri, requestingUser) =>
       nodePathGetAdminRequest(iri, requestingUser)
-    case ListRootNodeCreateRequestADM(createRootNode, requestingUser, apiRequestID) =>
+    case ListRootNodeCreateRequestADM(createRootNode, _, apiRequestID) =>
```

Comment:
Interesting, does it mean that we don't need the parameter `requestingUser` at all? (Also goes for all other cases in this file)

Reply:
yes, it seems so.